CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 2.7.0-beta1
 - [Added support to collect Perf Counters for .NET Core Apps if running inside Azure WebApps] (https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/889)
+- [Add operation details for HTTP and SQL operation to the dependency telemetry.](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/587)
```

Comment:
If possible create a new issue in gh, which contain detail about the specific fix. The original issue is quite long discussion, and it'll be hard to figure out which part from that is being implemented here.

Reply:
Also helpful if an example TelemetryInitializer can be provided which utilizes this new feature.

Reply:
Either is fine. Inline in a GH issue is easier to find, in my opinion though.

Reply:
K.  I'll code up a sample as I was intending to use this feature and include it in a new GH issue.

Reply:
Here is a consolidated issue with a code sample for intended use:
https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/900